pkg/kubelet/kubelet.go

```diff
@@ -1898,6 +1900,23 @@ func (kl *Kubelet) SyncPod(ctx context.Context, updateType kubetypes.SyncPodType
 		}
 	}
 
+	// If pod-level resources are set, CPU and Memory managers alignment is skipped,
+	// an event is surfaced to inform the user when skipped in the corresponding pod.
```

Comment:
We do not want to skip alignment for all cases. 
- If the container-level resources are set (regardless of pod-resources being set), align the resources at container-level
- If only pod-level resources are set, then skip the alignment

Reply:
Synced offline with @ndixita

Due to the QoS being determined by the pod level resources when specified in the pod spec, CPU and Memory managers alignment will be skipped for all cases when using pod level resources. Taking in consideration if containers specify request and limit for CPU and Memory, regardless of pod level resources being set or not, would make the QoS behavior inconsistent and create edge scenarios which could produce not intended behaviors.